src/components/body/Body.jsx

```diff
@@ -1,8 +1,22 @@
 import PropTypes from 'prop-types'
+import styles from './Body.module.scss'
+import Item from './Item'
+import data from '../../mocks/youtube-videos-mock'
 
 function Body(props) {
+  const {items} = data
+  console.log(items)
+
   return (
-    <div>Body</div>
+    <div className={styles.body}>
+      {items.map(({etag, snippet:{channelTitle, description, thumbnails}}) => {
+        return <Item 
+          key={etag} 
+          title={channelTitle} 
+          description={description} 
+          thumbnails={thumbnails} />
+      })}
+    </div>
   )
 }
 
```

Comment:
Missing empty line at the end of the file https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b